common/src/main/java/com/gregtechceu/gtceu/common/data/GTItems.java

```diff
@@ -662,6 +662,55 @@ public static ICustomDescriptionId cellName() {
             .onRegister(compassNodeExist(GTCompassSections.COVERS, "pump", GTCompassNodes.COVER))
             .register();
 
+    public static ItemEntry<ComponentItem> ELECTRIC_PUMP_UHV = GTCEu.isHighTier() ? REGISTRATE.item("uhv_electric_pump", ComponentItem::create)
+            .lang("UHV Electric Pump")
+            .onRegister(attach(new CoverPlaceBehavior(GTCovers.PUMPS[8])))
+            .onRegister(attach(new TooltipBehavior(lines -> {
+                lines.add(Component.translatable("item.gtceu.electric.pump.tooltip"));
+                lines.add(Component.translatable("gtceu.universal.tooltip.fluid_transfer_rate", 1280 * 64 * 64 * 4 / 20));
```

Comment:
Transfer rates need to be adjusted in the tooltip

Reply:
it's like this on 1.12 too, but sure, I'll do it